cmd/vmssrollout/vmssrollout.go

```diff
@@ -101,6 +112,78 @@ func readFile(path string) ([]byte, error) {
 	return ioutil.ReadFile(path)
 }
 
+func getScript(newPath, oldPath string) (*string, error) {
+	newScript, err := readScriptFromTemplate(newPath)
+	if err != nil {
+		return nil, err
+	}
+	oldScript, err := readScriptFromTemplate(oldPath)
+	if err != nil {
+		return nil, err
+	}
+	if equality.Semantic.DeepEqual(newScript, oldScript) {
+		return nil, nil
+	}
+	return newScript, nil
+}
+
+func readScriptFromTemplate(path string) (*string, error) {
```

Comment:
Reading from ARM to a compute.VirtualMachineScaleSet is ugly, may be simpler to walk raw maps.